node/node.go

```diff
@@ -89,6 +90,8 @@ const (
 	httpPortName    = constants.AppName + "-http"
 
 	ipResolutionTimeout = 30 * time.Second
+
+	apiNamespace = constants.PlatformName + metric.NamespaceSeparator + "api"
```

Comment:
here, I don't understand why we add `constants.PlatformName` to the metric.

Reply:
all of the metrics exported by avalanchego start with the `avalanche_` prefix. This is where we're adding that prefix.